recipes/cfflib/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cfflib" %}
+{% set version = "2.0.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "748043f2064079088d57a35074cf6406c436e2537e79a500330e654aadc19ac2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: true  # [py3k]
```

Comment:
you might consider putting this as a python version constraint `python <3`

Reply:
Ok like this skip: true #[python <3] ??

Reply:
more like this https://github.com/conda-forge/staged-recipes/pull/5654/files#diff-302a57ae4c9a65ce152a1f3d6a7fcf63R27 (you will be able to remove the skip entirely)